src/apps/watchfaces/OswAppWatchface.cpp

```diff
@@ -139,6 +142,9 @@ void OswAppWatchface::loop() {
     if (hal->btnHasGoneDown(BUTTON_2)) {
         hal->decreaseBrightness(25);
     }
+    if (hal->btnIsDoubleClick(BUTTON_2)) {
+        OswAppWatchface::settingDisplayMainWatchface("0");
```

Comment:
If this looks good, I want to apply it to all watchfaces. Also, there must be a way to get the current watchface index, can anyone tell me? It's a hard-coding method right now.

Reply:
https://en.cppreference.com/w/cpp/string/byte/strtol

https://en.cppreference.com/w/cpp/string/basic_string/to_string